code/game/atoms.dm

```diff
@@ -793,10 +792,15 @@
 /**
   * Respond to an emag being used on our atom
   *
-  * Default behaviour is to send COMSIG_ATOM_EMAG_ACT and return
+  * Args:
+  * * mob/user: The mob that used the emag. Nullable.
+  * * obj/item/card/emag/emag_card: The emag that was used. Nullable.
+  *
+  * Returns:
+  * TRUE if the emag had any effect, falsey otherwise.
   */
-/atom/proc/emag_act()
-	SEND_SIGNAL(src, COMSIG_ATOM_EMAG_ACT)
+/atom/proc/emag_act(mob/user, obj/item/card/emag/emag_card)
+	return (SEND_SIGNAL(src, COMSIG_ATOM_EMAG_ACT, user, emag_card))
```

Comment:
``SHOULD_CALL_PARENT``?

Reply:
Not many other acts ever use ..(), so I don't see why it should call parent. Will set to false.